packages/aws-cdk-lib/custom-resources/README.md

```diff
@@ -923,3 +926,22 @@ new ses.ReceiptRuleSet(app, 'RuleSet', {
   dropSpam: true,
 });    
 ```
+
+### Setting Lambda Runtimes
+
+The following example configures the custom resource lambda runtime to `PYTHON_3_12`:
```

Comment:
Can you update this to mention that the Lambda runtimes will only be set within a given runtime family? E.g. `addLambdaRuntime(Python.whatever)` will only modify Python resources, it won't modify Node ones. Also describe the use case of this; something like "you can use `addLambdaRuntime(lambdaRuntime)` to set every custom resource vended by the CDK to use the specified lambda runtime, provided that the custom resource's runtime is the same family as the one you specified."